### ISSUE TITLE
Unable to use v4l2loopback

### ISSUE THREAD

**Issue by cubit9-danielpeyer** [google-coral/edgetpu#520]
### Description

I can successfully install v4l2loopback-dkms and v4l2loopback-v4l2loopback-utils with `sudo apt install v4l2loopback-dkms v4l2loopback-utils`. But if I try to use it, like `sudo modprobe v4l2loopback devices=2`, it gives me an error like this `modprobe: FATAL: Module v4l2loopback not found in directory /lib/modules/4.14.98-imx`.

How can we fix that?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Operating System

Mendel Linux

### Coral Device

Dev Board

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
@cubit9-danielpeyer can you please share the output below commands. Thanks!
```

mendel@orange-horse:~$ ls /lib/modules/4.14.98-imx/
build  extra  kernel  modules.alias  modules.alias.bin  modules.builtin  modules.builtin.bin  modules.dep  modules.dep.bin  modules.devname  modules.order  modules.softdep  modules.symbols  modules.symbols.bin
mendel@orange-horse:~$ ls /lib/modules/4.14.98-imx/extra
v4l2loopback.ko  wlan.ko
```
<details>
```

mendel@orange-horse:~$ sudo apt install v4l2loopback-dkms v4l2loopback-utils
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  gstreamer1.0-tools
The following NEW packages will be installed:
  gstreamer1.0-tools v4l2loopback-dkms v4l2loopback-utils
0 upgraded, 3 newly installed, 0 to remove and 2 not upgraded.
Need to get 1,140 kB of archives.
After this operation, 1,346 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 https://mendel-linux.org/apt/eagle-bsp-enterprise eagle/main arm64 gstreamer1.0-tools arm64 1.14.4+imx-3 [1,086 kB]
Get:2 https://mendel-linux.org/apt/eagle eagle/main arm64 v4l2loopback-dkms all 0.12.1-1 [34.6 kB]
Get:3 https://mendel-linux.org/apt/eagle eagle/main arm64 v4l2loopback-utils all 0.12.1-1 [20.0 kB]
Fetched 1,140 kB in 1s (1,692 kB/s)           
Selecting previously unselected package gstreamer1.0-tools.
(Reading database ... 54549 files and directories currently installed.)
Preparing to unpack .../gstreamer1.0-tools_1.14.4+imx-3_arm64.deb ...

Progress: [  0%] [.....................................................................................................................................................................................................................] 
Unpacking gstreamer1.0-tools (1.14.4+imx-3) ...........................................................................................................................................................................................] 

Selecting previously unselected package v4l2loopback-dkms..............................................................................................................................................................................] 
Preparing to unpack .../v4l2loopback-dkms_0.12.1-1_all.deb ...

Unpacking v4l2loopback-dkms (0.12.1-1) ...#########################....................................................................................................................................................................] 

Selecting previously unselected package v4l2loopback-utils.########################....................................................................................................................................................] 
Preparing to unpack .../v4l2loopback-utils_0.12.1-1_all.deb ...

Unpacking v4l2loopback-utils (0.12.1-1) ...########################################################....................................................................................................................................] 

Setting up v4l2loopback-dkms (0.12.1-1) ...#########################################################################...................................................................................................................] 

Loading new v4l2loopback-0.12.1 DKMS files...#######################################################################################...................................................................................................] 
Building for 4.14.98-imx
Building initial module for 4.14.98-imx
Done.

v4l2loopback.ko:
Running module version sanity check.
 - Original module
   - No original module exists within this kernel
 - Installation
   - Installing to /lib/modules/4.14.98-imx/extra/

depmod...

DKMS: install completed.

Setting up gstreamer1.0-tools (1.14.4+imx-3) ...#####################################################################################################..................................................................................] 

Progress: [ 69%] [###################################################################################################################################################..................................................................] 
Setting up v4l2loopback-utils (0.12.1-1) ...#########################################################################################################################################..................................................] 

Progress: [ 85%] [####################################################################################################################################################################################.................................] 
Processing triggers for man-db (2.8.5-2) ...##########################################################################################################################################################################.................] 

```
</details>

**Comment by cubit9-danielpeyer** [comment]
mendel@vexing-tang:~$ sudo apt install v4l2loopback-dkms v4l2loopback-utils
Reading package lists... Done
Building dependency tree       
Reading state information... Done
v4l2loopback-dkms is already the newest version (0.12.1-1).
v4l2loopback-utils is already the newest version (0.12.1-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

mendel@vexing-tang:~$ ls /lib/modules/4.14.98-imx/
build  kernel         modules.alias.bin  modules.builtin.bin  modules.dep.bin  modules.order    modules.symbols
extra  modules.alias  modules.builtin    modules.dep          modules.devname  modules.softdep  modules.symbols.bin

mendel@vexing-tang:~$ ls /lib/modules/4.14.98-imx/extra
wlan.ko

**Comment by cubit9-danielpeyer** [comment]
@hjonnala Thanks for your help. I figured out what I missed. 
First I removed the two packages (`sudo apt remove v4l2loopback-dkms v4l2loopback-utils`) and then installed them again (`sudo apt install v4l2loopback-dkms v4l2loopback-utils`). 
If I had looked more closely to the output of the installation, the reason would be clear.

...
**_Module build for kernel 4.14.98-imx was skipped since the
kernel headers for this kernel does not seem to be installed._**
...
The kernel headers are missing!

So, install that package with `sudo apt-get install linux-headers-4.14.98-imx` and reinstall `v4l2loopback-dkms v4l2loopback-utils` solved my problem.
